.travis.yml

```diff
@@ -33,41 +33,50 @@ matrix:
   - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
   - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
-  - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=yes COLUMNS=80
   - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
-  - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=yes COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=dev SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=dev SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=dev SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=dev SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
   allow_failures:
   - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
   - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
-  - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
   - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
-  - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=dev SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=dev SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=dev SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=dev SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
 
 before_install:
 - docker pull mtmiller/octave:${OCT}
-- docker run --name=oc --detach --init --env=LC_ALL=C.UTF-8 --env=PYTHON_VERSION=${PYTHON_VERSION} --volume=$PWD:/octsympy:z mtmiller/octave:${OCT} sleep inf
+- docker run --name=oc --detach --init --env=LC_ALL=C.UTF-8 --env=PYTHON_VERSION=${PYTHON_VERSION} --env=PYTHON=python${PYTHON_VERSION} --volume=$PWD:/octsympy:z mtmiller/octave:${OCT} sleep inf
 
 install:
 - docker exec oc pip${PYTHON_VERSION} install sympy==${SYMPY_VER}
 - if [ "x$DOCTEST" = "xyes" ]; then
       docker exec oc octave --eval "pkg install -forge doctest";
   fi
 - if [ "x$PYTHONIC" = "xyes" ]; then
+      docker exec oc octave --eval "pkg install https://octave-pythonic-downloads.s3.amazonaws.com/releases/octave-pythonic-0.0.1.tar.gz";
+  fi
+- if [ "x$PYTHONIC" = "xdev" ]; then
       docker exec oc octave --eval "pkg install https://gitlab.com/mtmiller/octave-pythonic/-/archive/master/octave-pythonic-master.tar.gz";
   fi
+- if [ "x$PYTHONIC" = "xno" ]; then
+      docker exec oc octave --eval "f=fopen('~/.octaverc','a');fprintf(f,'#empty\n');fclose(f)";
+  else
+      docker exec oc octave --eval "f=fopen('~/.octaverc','a');fprintf(f,'pkg load pythonic\n');fclose(f)";
+  fi
```

Comment:
@mtmiller can you help with this part?  Now that Pythonic is used by default, we only need to load the package and it will be used.

1.  Does modifying (creating) `.octaverc` seem like a reasonable way to do that?

2.  Surely it should be something like `docker exec oc echo "pkg load pythonic" >> ${HOME}/.octaverc`.  But its not bash, and that actually calls `/bin/echo`...  Should it be something like `docker exec oc bash ...`"?  I gave up and just used Octave.

Reply:
Sure, I'll take a look

Reply:
eh?  That seemed to complain about `${HOME}` undefined or something when I did it...  Anyway looks much to me!